randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt

```diff
@@ -195,8 +197,10 @@ Extra - asset_id: collision_camera_004
           Any of the following:
               # Reaching the ledge
               Single-wall Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
-              # Freeze the Gullugg
+              # Freeze the Gullugg - TODO: get rid of template
               Stand on Frozen Enemy (Beginner) and Wall Jump (Beginner) and Fully Freeze Enemy
+              # Get up with HJ
+              High Jump Boots or Super Jump (Intermediate)
```

Comment:
This path is wrong but impossible anyway. Remove

Reply:
Oh, I added it to the wrong connection, whoops